keys.go

```diff
@@ -260,6 +269,41 @@ func (c *Client) SetKeyRing(ctx context.Context, idOrAlias, newKeyRingID string)
 	return &response.Keys[0], nil
 }
 
+// CreateKeyWithPolicyOverride creates a new KP key with given policy details
+func (c *Client) CreateKeyWithPolicyOverride(ctx context.Context, name string, expiration *time.Time, extractable bool, aliases []string, policy Policy) (*Key, error) {
+	return c.CreateImportedKeyWithPolicyOverride(ctx, name, expiration, "", "", "", extractable, aliases, policy)
+}
+
+// CreateImportedKeyWithPolicyOverride creates a new Imported KP key from the given key material and with given DualAuthDelete policy details
+func (c *Client) CreateImportedKeyWithPolicyOverride(ctx context.Context, name string, expiration *time.Time, payload, encryptedNonce, iv string, extractable bool, aliases []string, policy Policy) (*Key, error) {
+	key := c.createKeyTemplate(ctx, name, expiration, payload, encryptedNonce, iv, extractable, aliases, AlgorithmRSAOAEP256)
+
+	key.Rotation = policy.Rotation
+	key.DualAuthDelete = policy.DualAuth
```

Comment:
I think it will be good if we first check for nil and then do the assignment.
```suggestion
        if policy.Rotation != nil {
	       key.Rotation = policy.Rotation
	}
	if policy.DualAuth != ni {
	       key.DualAuthDelete = policy.DualAuth
	}
```

Reply:
Yeah, initially I added the check but later I found out that even if they(policy.Rotation and policy.DualAuth) are 'nil' the functionality works as intended as we would store nil inside key.Rotation and key.DualAuthDelete. So the check seemed redundant.